keyserver/server_test.go

```diff
@@ -77,14 +87,53 @@ func startServer(t *testing.T, kvdb kv.DB, epDeadline Timestamp, policiesPath st
 
 func TestServerStartStop(t *testing.T) {
 	utils.WithDB(func(db kv.DB) {
-		_, teardown := startServer(t, db, 60, "")
+		_, teardown := startServer(t, db, 60, true, "")
 		defer teardown()
 	})
 }
 
+func TestResolveAddresses(t *testing.T) {
+	dir, teardown := testutil.CreateTLSCertForTest(t)
+	defer teardown()
+
+	// test TCP network
+	addr := &Address{
+		Address:     "tcp:" + testutil.PublicConnection,
```

Comment:
I think the latest is here: https://github.com/coniks-sys/coniks-go/pull/130/files#diff-d24d2023b15c72fdfbc6fec8561dcf0dR55

> Maybe just change testutil.PublicConnection to include the protocol?

It is also being used by `testutil.NewTCPClient`. Yes, it should be a constant instead.